tests/serialisation_tests/test_serialisation.py

```diff
@@ -317,22 +317,21 @@ def test_serialise_data_region_filter(seeded_test_client, additional_test_data):
     when ITL region is passed, projects should be filtered by ITL region and any parent programmes with entirely
     filtered out child projects should not be returned.
     """
-    itl_regions = {ITLRegion.SouthWest}
-    test_query_region = download_data_base_query(itl_regions=itl_regions)
+    itl1_regions = [GeospatialDim.query.filter(GeospatialDim.postcode_prefix == "BS").one().itl1_region_code]
+    test_query_region = download_data_base_query(itl1_regions=itl1_regions)
```

Comment:
would this be clearer to read if you didn't instantly put the itl1_regions var into a one item list? As then you can set this param like `itl_regions=[itl_regions]`, and then later in the code you don't need to 0 index itl_regions to get the entity?